src/i18n/zh_CN.po

```diff
@@ -0,0 +1,223 @@
+msgstr "命令 '%s' 由下列软件包提供:"
+
+#: lookup.cpp:171
+#, c-format, boost-format
+msgid "A similar command to '%s' is provided by the following packages:"
+msgstr "类似于 '%s' 的命令由下列软件包提供:"
+
+#: package.cpp:47
+msgid "not a file: "
+msgstr "不是文件: "
+
+#: package.cpp:68
+msgid "this is not a valid package file: "
+msgstr "这不是一个有效的软件包文件: "
+
+#: package.cpp:111
+#, c-format, boost-format
+msgid "could not read file list from: %s"
+msgstr "无法从 %s 中读取文件列表"
+
+#: package.cpp:123
+#, c-format, boost-format
+msgid "error while closing archive: %s"
+msgstr "关闭归档 %s 时出错"
+
+#: populate.cpp:59
+#, c-format, boost-format
+msgid "       *** %s %s ***                                                  \n"
+msgstr "       *** %s %s ***                                                  \n"
+
+#: populate.cpp:61
+msgid "Usage:                                                                \n"
+msgstr "用法:                                                                \n"
+
+#: populate.cpp:62
+msgid "   cnf-populate -p <path> ( -c <catalog> | -m ) [ -d <path> ]         \n"
+msgstr "   cnf-populate -p <路径> ( -c <类别> | -m ) [ -d <路径> ]         \n"
+
+#: populate.cpp:63 populate.cpp:67
+msgid "                                                                      \n"
+msgstr "                                                                      \n"
+
+#: populate.cpp:64
+msgid "Options:                                                              \n"
+msgstr "选项:                                                              \n"
+
+#: populate.cpp:65
+msgid " --help            -? -h     Show this help and exit                  \n"
+msgstr " --help            -? -h     显示该帮助并退出                  \n"
+
+#: populate.cpp:66
+msgid " --verbose         -v        Display verbose output                   \n"
+msgstr " --verbose         -v        显示额外输出                   \n"
+
+#: populate.cpp:68
+msgid " --package-path    -p        Set the path containing the packages     \n"
+msgstr " --package-path    -p        设置包含该软件包的路径     \n"
+
+#: populate.cpp:69
+msgid " --catalog         -c        Set the catalog name to index (e.g. core)\n"
+msgstr " --catalog         -c        设置用于索引的类别名 (例如 core)\n"
+
+#: populate.cpp:70
+msgid " --mirror          -m        Scan mirror structure and detect catalogs\n"
+msgstr " --mirror          -m        扫描镜像结构并且检测类别\n"
+
+#: populate.cpp:71
+msgid " --truncate        -t        Truncate the catalog before indexing     \n"
+msgstr " --truncate        -t        在索引前截短类别     \n"
+
+#: populate.cpp:72
+#, c-format, boost-format
+msgid ""
+" --database-path   -d        Customize the database lookup path       \n"
+"                             default is %s                            \n"
+msgstr ""
+" --database-path   -d        自定义数据库查找路径\n"
+"                             默认为 %s                            \n"
+
+#: populate.cpp:139
+#, c-format, boost-format
+msgid "Not a valid package path: %s"
+msgstr "不是一个有效的软件包路径: %s"
+
+#~ msgid "       *** %s %s ***\n"
```

Comment:
is this commented block supposed to be here? I am not that familiar with the file format though ...